__tests__/integration.test.js

```diff
@@ -1,30 +1,102 @@
 const app = require("../app");
 const request = require("supertest");
-const seed  = require("../db/seeds/seed")
-const db = require("../db/connection")
-const testData = require("../db/data/test-data/index")
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const testData = require("../db/data/test-data/index");
+const endpoints = require("../endpoints.json");
 
 beforeEach(() => {
-    return seed(testData)
-})
-
-afterAll(()=> {
-    db.end()
-})
-
-describe('getTopics', () => {
-    it('should return a 200 status code', () => {
-        return request(app).get("/api/topics").expect(200)
-    });
-    it('should return an array of topic objects, each of which should have the following properties: slug, description) ', () => {
-        return request(app).get("/api/topics").then((res) => {
-            expect(Array.isArray(res.body.topics)).toBe(true)
-            expect(res.body.topics).toHaveLength(3)
-            res.body.topics.forEach((topic, index) => {
-                expect(typeof res.body.topics[index]).toBe('object')
-                expect(topic).hasOwnProperty('slug')
-                expect(topic).hasOwnProperty('description')
-            });
-        })
-    });
-});
\ No newline at end of file
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("getTopics", () => {
+  it("should return a 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  it("should return an array of topic objects, each of which should have the following properties: slug, description) ", () => {
+    return request(app)
+      .get("/api/topics")
+      .then((res) => {
+        expect(res.body.topics).toHaveLength(3);
+        res.body.topics.forEach((topic, index) => {
+          expect(typeof res.body.topics[index]).toBe("object");
+          expect(topic).hasOwnProperty("slug");
+          expect(topic).hasOwnProperty("description");
+        });
+      });
+  });
+});
+
+describe("getAPI", () => {
+  it("should return an object describing all the available endpoints on your API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        const testObject = endpoints;
+        const endpointResult = res.body;
+        expect(testObject).toEqual(endpointResult);
+      });
+  });
+});
+
+describe("getArticleByID", () => {
+  it("should return correct article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        const testArticle = res.body.article[0];
```

Comment:
We should not be expecting an array here, res.body.article should be the single article object